doc/getting_started.rst

```diff
@@ -69,7 +69,7 @@ newly transformed sample matrix ``X``::
   >>> from sklearn.preprocessing import StandardScaler
   >>> X = [[0, 15],
   ...      [1, -10]]
-  >>> StandardScaler().fit(X).transform(X)
+  >>> StandardScaler().fit(X).transform(X) # scale data based on the standard score
```

Comment:
a 'score' has a specific meaning in scikit-learn (e.g. it means r2 score or accuracy score)

how about 

> scale data according to computed scaling values

(feel free to edit and make sure it fits in the line length, if it doesn't we can add the comment above)

Reply:
Okay. I will use the suggestion. Thanks!